### DISCUSSION TITLE
2 New ideas I thought of

### DISCUSSION THREAD

**Discussion by DetectiveIdiot** [Malcolm-Q/LC-LateGameUpgrades#520] [Ideas]
More pockets or Bigger backpack (Whatever you want to name it): Gives player 1 extra inventory slot per upgrade, you can give them how many you want, (like 4 upgrades if you want, you can make them cost whatever you want, but I was thinking 500, 750, 1000, 1250 or 1500 is what I'm thinking) late game moons like Titan and Artifice would have a lot of scrap players to collect or gather before 12AM, the ability to carry more would be useful for people who go to late game moons.

Early bird: Ship lands early, giving players more time to collect scrap (I hope it is possible for you to create), with the mod "AfterNight" by woah25_discord, the earliest you can go (Or try to go for) is 6:00AM, because every time it passes 12:00AM, it jumps to 6:00AM. This is useful for players who are in the late game because how much more time they get (I know there's already an upgrade that slows down time, but for some reason, it jumps from 8:00AM to 8:30AM when the ship lands).

**Comment by WhiteSpike** [comment]
Unless I'm misunderstanding something, 2nd seems like the upgrade "landing thrusters" where it speeds up the ship landing on the moon, no?

**Reply by DetectiveIdiot** [reply]
I look into it. It speeds up the ship's landing/taking off, but it doesn't land early. I think.

**Reply by maximo4** [reply]
"Quantum disruptor" extends the time on a moon.
Is that what you're looking for?
